src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -75,8 +86,11 @@ public bool HasContainer
 
 		public abstract void NativeArrange(Rectangle frame);
 
+		partial void ConnectingHandler(NativeView? nativeView);
+
 		private protected void ConnectHandler(NativeView? nativeView)
 		{
+			ConnectingHandler(nativeView);
 		}
```

Comment:
Is this the right value to be watching? Should we be tracking the actual frame bounds set by the layout engine?